packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -53,24 +63,49 @@ pub fn rest_api_router<P: GraphPool>(store: Arc<P>) -> Router {
     merged_routes
         // Make sure extensions are added at the end so they are made available to merged routers.
         .layer(Extension(store))
-        .route(
-            "/api-doc/openapi.json",
+        .nest("/api-doc", Router::new().route(
+            "/openapi.json",
             get({
                 let doc = open_api_doc;
                 move || async { Json(doc) }
-            }),
+            })).route("/models/*path", get(static_schemas)),
         )
 }
 
+#[allow(
+    clippy::unused_async,
+    reason = "This route does not need async capabilities, but axum requires it in trait bounds."
+)]
+async fn static_schemas<'a>(Path(path): Path<String>) -> impl IntoResponse {
+    let path = path.trim_start_matches('/');
+
+    match STATIC_SCHEMAS.get_file(path) {
+        None => Response::builder()
+            .status(StatusCode::NOT_FOUND)
+            .body(body::boxed(Empty::new()))
+            .expect("Could not create 404 response"),
+        Some(file) => Response::builder()
+            .status(StatusCode::OK)
+            .header(
+                header::CONTENT_TYPE,
+                HeaderValue::from_str("application/json")
+                    .expect("Could not create header value for found static file"),
+            )
+            .body(body::boxed(Full::from(file.contents())))
+            .expect("Could not serve static file"),
+    }
+}
+
 #[derive(OpenApi)]
 #[openapi(
         tags(
             (name = "Graph", description = "HASH Graph API")
         ),
-        modifiers(&MergeAddon)
+        modifiers(&MergeAddon, &ExternalRefAddon)
     )]
 struct OpenApiDocumentation;
 
+/// Addon to merge multiple API Documentations together.
```

Comment:
What is an "API Documentation" is there a Rust struct or something we can intra-doc link to here?

Reply:
Addressed in 757c5e64dad3688a6fc91ef2da3fe1af7750db26

Reply:
I'm asking because I didn't understand what it was 😁 (so for code comprehension rather than pub doc reasons haha)

Reply:
Thanks for clarifying